Makefile

```diff
@@ -1,23 +1,28 @@
-build:
+.PHONY: html
+html:
 	rm -fr public
 	hugo --theme=golangshow
 
+.PHONY: whitespace
+whitespace:
+	go run tools/strip-whitespace/main.go
+
+.PHONY: build
+build: html whitespace
```

Comment:
Why `build`, `html` and `whitespace` are separate targets? In what case we need them separately?